_guides/client/basic.md

```diff
@@ -11,6 +11,16 @@ extern crate hyper;
 extern crate tokio_core;
 ```
 
+Make sure you have these dependencies listed in your ```Cargo.toml```, else you may get errors. You may need 
+to check if these are the latest versions from [Crates.io](https://crates.io)
+
+```
```

Comment:
This code fence can have 'toml' put here, which will use different syntax highlighting, and also tell the tests that this isn't Rust code.